plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1007,6 +1008,18 @@ public void testNullSensitiveTopNPushdown()
         }
     }
 
+    @Test
+    public void testPredicateModuloPushdown()
```

Comment:
"testPredicatePushdownWithModulus"

however, maybe we can consider "arithmetics" as a single behavior, and not be more fine-grained, like @findepi did in https://github.com/trinodb/trino/blob/bd9c79f044d927de55624b71e85a1c80c8b8f1be/plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java#L709-L713

Reply:
this should result in removing/moving the test we have in PostgreSQL here

Reply:
renamed